README.md

```diff
@@ -61,6 +61,10 @@ A curated list of awesome inclusive events/projects/initiatives for women in the
 ### Javascript
 ([Go to top](#awesome4girls---))
 
+  - [Js4Girls](http://webschool.io/js4girls/) - Loc: Brazil - - *Loc:* <img src="https://upload.wikimedia.org/wikipedia/en/0/05/Flag_of_Brazil.svg" alt="Brazil" width="30"> - *Lang: PT-BR only*
+  
+  A Brazilian initiative to provide free javascript classes to women in a male-free environment.
```

Comment:
I found the problem. Here must have 4 spaces. I will add it on the fail description